fixtures/app/extensions/customer-accounts-ui/shopify.ui.extension.toml

```diff
@@ -0,0 +1,8 @@
+type = "customer_accounts_ui_extension"
+name = "customer-account-extension"
+
+extension_points = [
+  'CustomerAccount::FullPage::RenderWithin',
+]
+
+categories = ['returns']
```

Comment:
if you have returns enabled you will also need the `CustomerAccount::Returns::Initiate` extension point